src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -825,6 +825,9 @@ void DeformableConvolution::initSupportedPrimitiveDescriptors() {
 
     auto &weiDims = getInputShapeAtPort(WEI_ID).getDims();
     if (weiDims[1] == Shape::UNDEFINED_DIM || weiDims[0] == Shape::UNDEFINED_DIM ||
+        // 1. strict fallback, until devising of multigroup handling in common case
+        defConvAttr.group != 1 ||
+        // 2. common fallback, except specific n_group / n_channel combinations
         (defConvAttr.group != 1 && ((weiDims[1] % simd_w != 0)  // in_channels_per_gr !% simd_w
```

Comment:
Seems like "defConvAttr.group != 1" condition in this line is not needed anymore.

Reply:
Logically, second condition overwraps with first, but these two conditions corresponds different levels of multigroup support. If reference performance will be too low for some specific models in the future, but second channels/groups relation will be satisfied, we will add corresponding logic in jit-code, remove the first fallback and leave the second.